### ISSUE TITLE
SSD or Faster-RCNN

### ISSUE THREAD

**Issue by EscVM** [fizyr/keras-retinanet#739]
Hi,

I have one little question; is keras retinanet an SSD model with focal loss?

Thx

**Comment by hgaiser** [comment]
Retinanet is [retinanet](https://arxiv.org/pdf/1708.02002.pdf).

**Comment by hgaiser** [comment]
In that sense, yes.

**Comment by EscVM** [comment]
Ok! I had this doubt because I was searching for a good implementation of a Faster RCNN model and I found this [repository](https://github.com/yhenon/keras-frcnn). In the readme ther's written "This repo is now deprecated. Instead, use https://github.com/fizyr/keras-retinanet. Keras-retinanet is more accurate, has higher quality code and runs faster". Reading that it seemed to me that retinanet was a F-RCNN.

**Comment by hgaiser** [comment]
Ah, no it's a different architecture. It borrows some stuff from Faster-RCNN but they differ significantly. Most noticeably, it is 1 stage instead of 2 stage.

**Comment by EscVM** [comment]
Do you know any good implementation of a Faster-RCNN? It would be perfect made with TensorFlow/Keras.

**Comment by hgaiser** [comment]
To be honest, RetinaNet is an improvement over Faster-RCNN on many (all?) levels. I wouldn't use Faster-RCNN.

**Comment by EscVM** [comment]
I personally disagree! Faster-RCNN are for sure slower, but they can result in a better mAP.

**Comment by CUGfred** [comment]
are u sure about that faster-rcnn better mAP? i am search for an algorithm which get better mAP,could u please tell me something about tha?
